Cargo.toml

```diff
@@ -75,7 +75,7 @@ rusoto_sts = { version = "0.41.0", optional = true }
 
 # Tower
 tower = "0.1.1"
-tower03 = { package = "tower", version = "0.3" }
+tower03 = { package = "tower", version = "0.3", git = "https://github.com/tower-rs/tower", features = ["limit", "retry", "timeout", "util"] }
```

Comment:
Why the git dep?

Reply:
current release of 0.3 has some bounds issues that are fixed on master, we plan to release 0.4 soon with proper docs etc. So this is a place holder until then. The master version contains the layers we need as well that I up streamed.

Reply:
https://github.com/tower-rs/tower/issues/431